package.json

```
@@ -18,7 +18,8 @@
     "copy-templates": "rsync -av --exclude '*node_modules*' ./src/code-templates/ ./.dist/src/code-templates",
     "build:watch": "tsc --watch",
     "start:cli": "node ./.dist/bin/cli.js interactive",
-    "publish:build": "npm run build && npm publish ./.dist --access public"
+    "publish:build": "npm run build && npm publish ./.dist --access public",
+    "prepare": "husky install"
```

> Then how do we fix this? Maybe activate only one flag --initialize-git --apply-git-hook?

@goldbergyoni @jalvar53 We can fix it but only running `husky install` if `NODE_ENV` is different than `test`

(Need to check if works well for Windows as well 😄)
```suggestion
    "prepare": "test \"$NODE_ENV\" == \"test\" || husky install"
```